docs/specs/toc.yml

```diff
@@ -1232,3 +1232,39 @@ outputs:
   docs/a.json:
   docs/toc.json: |
     {"items":[{"toc_title":"TOC Ref","children":[{"toc_title":"Uid Ref","href":"a","monikers":[]}],"monikers":[]}]}
+---
+# uid reference in markdown toc using bad syntax(multiple inlines for one heading block)
+# the xref inline has highest priority, then we only retrieve the first one
```

Comment:
@superyyrrzz what is the behavior for v3 in this case? I feel like its some kind of error or warning

Reply:
v2 will error if toc.md has wrong syntax.

Reply:
ok, i will treat them as erros